### PR TITLE
Rename recordPosition into position

### PR DESCRIPTION
**Pull request by thomtrp**
As title

### CODE REVIEW

File: packages/twenty-server/src/database/typeorm-seeds/workspace/opportunity.ts [hunk 1]
```diff
@@ -32,7 +32,7 @@ export const seedOpportunity = async (
         closeDate: new Date(),
         probability: 0.5,
         stage: 'new',
-        recordPosition: 0,
+        pos: 0,
```
Comment (Weiko) [inline packages/twenty-server/src/database/typeorm-seeds/workspace/opportunity.ts:35]: position?

**Comment by charlesBochet** [comment]
@Weiko I don't think adding "record" in front position gives any extra information. It would be record.recordPosition
We don't have a position for custom object so far but we will be adding the field any time we create a custom object (like updatedAt, createdAt, name)

**Review by charlesBochet — APPROVED** [review]
I've added position to seeded companies, person and opportunities (for demo and dev)

**Comment by Weiko** [comment]
> @Weiko I don't think adding "record" in front position gives any extra information. It would be record.recordPosition
> We don't have a position for custom object so far but we will be adding the field any time we create a custom object (like updatedAt, createdAt, name)

Why record.recordPosition? In DB it will be company.recordPosition right?

**Review by Weiko — APPROVED** [review]

**Pull request merged**